### DISCUSSION TITLE
疯了，最近这个节点，乱跳！

### DISCUSSION THREAD

**Discussion by blacksparkler** [getlantern/lantern-forum-cn#475] [General]
一直在US HK SG 之间疯狂切换，且US和HK占绝大多数，可问题是新版PS用户，只能用SG啊！图都做不了，怎么办？收费的是不是得拿出收费软件的质量来呢？生气气！！！
![IMG_20190718_205027](https://github.com/getlantern/lantern-forum-cn/assets/168267707/485fc018-cc2e-48d7-8f97-bdb7edadc371)
![IMG_20190718_205046](https://github.com/getlantern/lantern-forum-cn/assets/168267707/d1ad7fad-ccac-44e6-8a90-93e4af0e2ab7)
![mmexport1568725199951](https://github.com/getlantern/lantern-forum-cn/assets/168267707/4983609e-28df-4b6e-a127-89c0e51c6dae)
![mmexport1592219295577](https://github.com/getlantern/lantern-forum-cn/assets/168267707/891d2b66-8f98-41a3-9f9d-91daeb38cd85)

**Comment by MaxMKMax** [comment]
@blacksparkler 
您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@blacksparkler 我想确认一下是不是因为节点乱跳所以导致网络不稳定呢?
请您息怒. 我会让技术团队知道猫猫怒了. 请您耐心等待我们修复. 谢谢您!

**Comment by keene06** [comment]
同样的问题，us和sg乱跳，也是只有sg可以连上网

**Reply by xiaobai2345** [reply]
更新到最新版本的试试吧

**Reply by MaxMKMax** [reply]
https://github.com/getlantern/lantern
@keene06 @blacksparkler 您们试试看从这里重新下载最新版本. 谢谢!

**Reply by carpediemcc** [reply]
@Mike777Khoo 我这之前只有连US才能用，现在也是节点乱跳，用你这链接下了所谓最新版本也没有用。刚好这几天去了趟外地，我发现在外地情况就好一些，虽然节点也会变但是词数少了，现在回家就完全用不了。而且应该不是网络运营商的问题，因为我用手机热点就能正常使用（虽然还是会卡）。这问题到底啥时候能解决，或者你们技术团队给分析一下到底是我电脑网络设置有问题还是你们服务器不稳定了？

**Reply by MaxMKMax** [reply]
@carpediemcc  您好. 服务器跳来跳去是因为系统会自动选最稳定的服务器给你们. 但是估计调过头了反而弄巧反拙. 明白了, 我会再次帮你们反馈的, 放心!

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。

**Reply by carpediemcc** [reply]
@Mike777Khoo {72130961}，现在的问题主要是我想搞清楚到底是你们服务器卡了还是我网络设置有问题，因为不用手机热点连的话有时候虽然连上了但是速度很慢，不用路由器直接接网线也是一样，具体能连什么服务器目前不重要，别卡就行。

**Reply by MaxMKMax** [reply]
@carpediemcc 好的. 谢谢您的票号. 我帮您反馈一下.

**Comment by jean-reno** [comment]
只要连在SG就没事 ，一卡，回头查看蓝灯 就发现蓝灯自动连接在了US 或者DE  这时候 就啥也打不开了 只能卡在那里  等蓝灯自动转到SG  才会不卡 也是无语了

**Reply by MaxMKMax** [reply]
@jean-reno 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。他们查了log才能找出问题出现在哪里。

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码.

**Comment by MaxMKMax** [comment]
@jay-418 @NolanRy I added the new log {72130961} to the previous ticket that we raised in the previous meeting. However, the server issues have impacted quite a lot of users. 

The first user (blacksparkler) feedback that he is only able to use the SG server to operate the photoshop software. The other server is not working.

**Reply by jay-418** [reply]
Thanks @Mike777Khoo! The team is working on this.

**Comment by MaxMKMax** [comment]
@blacksparkler 您好. 请问您已经提交好报告了吗? 请您提供我 8 位数票号. 谢谢哦!
